src/containerapp/HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+
+0.1.0
+++++++
+* Initial release.
```

Comment:
Nit: Can we call this "Initial release for container app support with Microsoft.App RP